central/graphql/resolvers/loaders/images_v2.go

```diff
@@ -0,0 +1,172 @@
+package loaders
+
+import (
+	"context"
+	"reflect"
+
+	"github.com/pkg/errors"
+	"github.com/stackrox/rox/central/imagev2/datastore"
+	imagesView "github.com/stackrox/rox/central/views/images"
+	v1 "github.com/stackrox/rox/generated/api/v1"
+	"github.com/stackrox/rox/generated/storage"
+	"github.com/stackrox/rox/pkg/concurrency"
+	searchPkg "github.com/stackrox/rox/pkg/search"
+	"github.com/stackrox/rox/pkg/sync"
+)
+
+var imageV2LoaderType = reflect.TypeOf(storage.ImageV2{})
+
+func init() {
+	RegisterTypeFactory(reflect.TypeOf(storage.ImageV2{}), func() interface{} {
+		return NewImageV2Loader(datastore.Singleton(), imagesView.Singleton())
+	})
+}
+
+// NewImageV2Loader creates a new loader for image data. If postgres is enabled, this loader holds images without scan data—components and vulns.
```

Comment:
nit:  remove the "If postgres is enabled," part.  The rest of that sentence is always true as Postgres is our only option.